packages/reapit-connect-management/src/components/ui/offices/__tests__/offices-groups-tab.test.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react'
+import { shallow } from 'enzyme'
+import OfficesGroupsTab, { OfficeGroupsContent, onPageChangeHandler } from '../offices-groups-tab'
+import { createBrowserHistory } from 'history'
+import Routes from '@/constants/routes'
+
+const locationMock = { pathname: '/offices/groups' }
+
+jest.mock('react-router', () => ({
+  ...jest.requireActual('react-router'),
+  useLocation: jest.fn(() => locationMock),
+}))
+
+const data = {
```

Comment:
Can you move into a stubs folder please and also add a type definition to the stub